components/spin/style/index.tsx

```diff
@@ -71,7 +71,7 @@ const genSpinStyle: GenerateStyle<SpinToken> = (token: SpinToken): CSSObject =>
           marginTop: -(token.spinDotSize / 2) - 10,
         },
 
-        [`> div > ${token.componentCls}-sm`]: {
+        '&-sm': {
```

Comment:
Why update this?

Reply:
Removing all of the css rules to illustrate the problem gives us this:

```typescript
const genSpinStyle: GenerateStyle<SpinToken> = (token: SpinToken): CSSObject => ({
  [`${token.componentCls}`]: {
    '&-nested-loading': {
      [`> div > ${token.componentCls}`]: {
        [`> div > ${token.componentCls}-sm`]: {},
        [`> div > ${token.componentCls}-lg`]: {},
      },
    },
  },
});
```

## Proposed solution 1

```typescript
const genSpinStyle: GenerateStyle<SpinToken> = (token: SpinToken): CSSObject => ({
  [`${token.componentCls}`]: {
    '&-nested-loading': {
      [`> div > ${token.componentCls}`]: {
        '&-sm': {},
        '&-lg': {},
      },
    },
  },
});
```

## Proposed solution 2

```typescript
const genSpinStyle: GenerateStyle<SpinToken> = (token: SpinToken): CSSObject => ({
  [`${token.componentCls}`]: {
    '&-nested-loading': {
      [`> div > ${token.componentCls}`]: {},
      [`> div > ${token.componentCls}-sm`]: {},
      [`> div > ${token.componentCls}-lg`]: {},
    },
  },
});
```